services/feed/feed.spec.js

```diff
@@ -34,44 +39,3 @@ describe("readFile FUNC TEST", () => {
     )
   })
 })
-
-describe("responseFormatter FUNC TEST", () => {
-  const start = moment("2016-06-15T00:00:00")
-  const end = moment("2020-07-15T23:59:59")
-  it("it should return formatted day wise grouped data", async () => {
-    const result = await service.responseFormatter(
-      groupedDataDaywise,
-      start,
-      end,
-      true
-    )
-    expect(typeof result).toBe("object")
-    expect(result).toContainEqual(
-      expect.objectContaining({
-        intervalStart: expect.anything(),
```

Comment:
Please dont user `anything`. Instead use proper validation of data and dataTypes